Project.toml

```diff
@@ -56,4 +61,5 @@ StatsBase = "0.33, 0.34"
 Tables = "1.10"
 TaylorIntegration = "0.15"
 TaylorSeries = "0.17"
+Test = "1"
```

Comment:
`Test` is no longer included in NEOs deps, so I think it's safe to remove this compat entry
```suggestion
```